.travis.yml

```diff
@@ -8,5 +8,5 @@ jobs:
   include:
     - script: npm run test-lib
     - script: npm run test-components
-    - script: npm run test-inflections-1
-    - script: npm run test-inflections-2
+    #- script: npm run test-inflections-1
```

Comment:
I have commented these tests - they are really heavy and I think should be in inflection-tables.
I think we could avoid them for now.